### DISCUSSION TITLE
how and when should I use meta-dump tool, please?

### DISCUSSION THREAD

**Discussion by wey-gu** [vesoft-inc/nebula#4221] [Q&A] [answered]
as titled :)

**Comment by HarrisChu** [comment, marked as answer]
meta-dump -path {meta_rocksdb_data}

**Reply by wey-gu** [reply]
When should we use it, please? to manually backup all data of meta?

**Reply by HarrisChu** [reply]
When we debug something.

===
> to manually backup all data of meta?

no, just display some important information, not all the data in metad.

**Reply by wey-gu** [reply]
Got it @HarrisChu, thanks so much, will give it a try!
